params_and_main.py

```diff
@@ -14,7 +14,7 @@
 import pathlib
 import os
 import warnings
-
+import albumentations as A
```

Comment:
For the future I think it would be good to not have to import albumentations here but in the train.py -> this could be useful for the json approach as we then have to only define 'strings' for the augmentations to be passed and the actual objects to be created in a different module (train.py) -> for now I think its ok like this, too, just as a note